src/components/GreyBG.js

```diff
@@ -27,58 +27,60 @@ export default styled.section`
   padding-top: 0;
   margin-bottom: 0;
 
-  ${breakpoint('smallTablet')`
-    margin-top: ${({ topOffset }) =>
-      remcalc((topOffset && topOffset * -1) || PADDING)};
-    margin-bottom: ${({ topOffset }) => remcalc(topOffset || -PADDING)};
-    &:after {
-      ${afterStyles}
-      height: ${({ topOffset }) =>
-        remcalc(topOffset ? topOffset * -1 : PADDING)};
-    }
-  `}
+  ${isNot('static')`
```

Comment:
`static` is meant to be a prop to disable the negative margins that currenlty the `GreyBG` applies by default. I visited some pages in the website and see why this is needed. So I thought a single prop could be a good solution to disable that behavior when needed.